source/documentation/rdf/datasets.md

```diff
@@ -57,8 +57,8 @@ with data read from URLs (files or from any other URL).
 
 The examples use the following prefixes:
 
-    @prefix ja:     <http://jena.hpl.hp.com/2005/11/Assembler#> .
-    @prefix rdf:    <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+    PREFIX ja:     <http://jena.hpl.hp.com/2005/11/ASSEMBLER#>
```

Comment:
Is it really `ASSEMBLER#` and not `Assembler#`?

Reply:
Done